pkg/csvmum/common.go

```diff
@@ -0,0 +1,36 @@
+package csvmum
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+)
+
+func getHeaderNamesToIndices(t reflect.Type) (map[string]int, error) {
+	headers := map[string]int{}
+
+	if t.Kind() != reflect.Struct {
+		return headers, fmt.Errorf("cannot get headers: not a struct")
+	}
+	for i := 0; i < t.NumField(); i++ {
```

Comment:
does `for i := range t.NumField()` work?

Reply:
I meant to do that!